webapp/channels/src/components/integrations/integration_option.tsx

```diff
@@ -11,27 +11,25 @@ type Props = {
     link: string;
 }
 
-export default class IntegrationOption extends React.PureComponent <Props> {
-    render() {
-        const {image, title, description, link} = this.props;
+const IntegrationOption = React.memo(({image, title, description, link}: Props) => {
```

Comment:
nit: We normally put the `React.memo` in the export line, because keeps this line cleaner, and this line is normally a more complicated one.